src/sysjson.go

```diff
@@ -15,10 +16,20 @@ var (
 	cert     = flag.String("cert", "", "TLS cert file")
 	key      = flag.String("key", "", "TLS key file")
 	password = flag.String("password", "", "Enable basic authentication")
+	from	=	flag.String("from", "", "Address to provide statistics to")
```

Comment:
I think this may be a bit confusing. We're not providing statistics *to* anything explicitly, we're getting statistics *from* sys.json. How about something like "CIDR whitelist for requests".